src/App.tsx

```diff
@@ -1,17 +1,45 @@
 import './App.scss';
-import Title from './components/Title';
-import Counter from './components/Counter';
+import React from 'react';
+import AppBar from './components/AppBar';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Home from './pages/Home/Homepage';
+import CS2 from './pages/CS2/CS2Page';
+import VALORANT from './pages/VALORANT/ValorantPage';
+import { ROUTES } from './constants/Routes';
 
 function App() {
   return (
-    <div className="App">
-      <Counter />
-      <Title title="Raman Rahinia" />
-      <div className="info">
-        <p>I like to play video games like</p> <a href={process.env.REACT_APP_CS_LINK}>CS2</a> or{' '}
-        <a href={process.env.REACT_APP_VALORANT_LINK}> VALORANT</a>
-      </div>
-    </div>
+    <BrowserRouter>
+      <AppBar
+        links={[
```

Comment:
you should not pass static data through props, props are needed for dynamic data.
you can configure it in AppBar component itself